src/behavior.ts

```diff
@@ -123,13 +127,24 @@ export class WordCloudBehavior {
         });
     }
 
+    private bindKeyboardEventToWords(): void {
+        this.behaviorOptions.wordsSelection.on("keydown", (event : KeyboardEvent, word: WordCloudDataPoint) => {
```

Comment:
multiselection with Ctrl

Reply:
check Shift behavior (should move the focus in the opposite direction)

Reply:
in selectWord function